freqtrade/exchange/exchange.py

```diff
@@ -576,7 +617,21 @@ def get_min_pair_stake_amount(self, pair: str, price: float,
         # The value returned should satisfy both limits: for amount (base currency) and
         # for cost (quote, stake currency), so max() is used here.
         # See also #2575 at github.
-        return max(min_stake_amounts) * amount_reserve_percent
+        return self._apply_leverage_to_stake_amount(
+            max(min_stake_amounts) * amount_reserve_percent,
+            leverage or 1.0
+        )
+
+    def _apply_leverage_to_stake_amount(self, stake_amount: float, leverage: float):
+        """
+        #TODO-lev: Find out how this works on Kraken and FTX
+        # * Should be implemented by child classes if leverage affects the stake_amount
```

Comment:
correct me if i'm wrong, but i think this comment is outdated (you did tests how this works for kraken / ftx).